.ci/seed_es_on_travis.sh

```diff
@@ -76,7 +76,21 @@ case "$ES_VERSION" in
       export SAMPLE_DATA_FILE="${ES7_SAMPLE_DATA_FILE}"
       ;;
 
-    "7.3.1")
+    "7.3.2")
+      export MAPPING_FILE=${ES7_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION-amd64.deb"
+      # overwrite SAMPLE_DATA_FILE to use the ES7-compliant data
+      export SAMPLE_DATA_FILE="${ES7_SAMPLE_DATA_FILE}"
+      ;;
+
+    "7.4.2")
+      export MAPPING_FILE=${ES7_MAPPING_FILE};
+      export ES_BINARY_URL="${ES5PLUS_ARCHIVE}/elasticsearch-$ES_VERSION-amd64.deb"
+      # overwrite SAMPLE_DATA_FILE to use the ES7-compliant data
+      export SAMPLE_DATA_FILE="${ES7_SAMPLE_DATA_FILE}"
+      ;;
+
+    "7.4.2")
```

Comment:
Whoops:
```suggestion
    "7.5.2")
```

Reply:
glad to have you reviewing my code again   @austin3dickey 😀